cpp/src/arrow/builder.h

```diff
@@ -18,6 +18,7 @@
 #pragma once
 
 #include <memory>
+#include <utility>
```

Comment:
Not sure why you had to change this header file?

Reply:
My mistake, this was checked in erroneously. I'll remove it